src/main/archetype/ui.frontend/src/main/webpack/site/styles/experiencefragment_footer.scss

```diff
@@ -0,0 +1,16 @@
+
+//== XF Footer style, used on page template
+
+footer.experiencefragment {
+    padding: .5em 1em;
+    border-top: 1px solid $color-foreground;
+
+    @media (prefers-color-scheme: dark) {
+        border-color: $color-foreground-dark;
+    }
+    
+    .cmp-separator hr {
```

Comment:
we recommend earlier that customers don't style element names, but we do so here, I think we can use `.cmp-separator__horizontal-rule`

Reply:
this also seems like a common reset for the separator base component rather than specific to the footer.